meta-xt-prod-cockpit-rcar-driver-domain/recipes-qt/examples/dual-render-3d-part.bb

```diff
@@ -0,0 +1,39 @@
+DESCRIPTION = "Qt Quick 3D demo cluster"
+
+LICENSE = "CLOSED"
+
+inherit systemd
+
+RDEPENDS_${PN} = " \
+         qtquick3d \
+         qtbase \
+         qtdeclarative \
+"  
+
+FILES_${PN} = " \
+    ${bindir}/dual-render-3d-part \
+    ${systemd_unitdir}/system/3d-part.service \
+    ${sysconfdir}/eglfs.conf \
+"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += " \
+	file://3d-part.service \
+	git://git@gitpct.epam.com:rec-inv/dual-render-3d-part.git;branch=main \
```

Comment:
Please replace `:` by `/` and append `;protocol=ssh`.
So this line has to be like (I have tested it)
```
git://git@gitpct.epam.com/rec-inv/dual-render-3d-part.git;branch=main;protocol=ssh \
```